content-services/content-sources-backend
# Fixes 4897: env_created field reset after updating template

## Summary

Fixes query that sets the `rhsm_environment_created` field. Without this change, any subsequent template update task would set all the other templates' `rhsm_environment_created` fields to false.

(This is not testable in ephemeral because it depends on candlepin)

## Testing steps

1. Create a template. `rhsm_environment_created` should be false initially in the response
2. Fetch the template. `rhsm_environment_created` should be true
3. Create another template and then list them after a couple seconds to let the update-template-content task complete
4. Both templates should have their `rhsm_environment_created` fields set to true
